### PR TITLE
[release-1.9] stop publishing latest tags

### PR DESCRIPTION
**Pull request by shamsher31**
Part of https://github.com/istio/istio/issues/29490#

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: example/manifest.yaml [hunk 1]
```diff
@@ -40,17 +40,17 @@ dependencies:
...
+    branch: release-1.9
     goversionenabled: true
   gogo-genproto:
     git: https://github.com/istio/gogo-genproto
-    branch: master
+    branch: release-1.9
   test-infra:
     git: https://github.com/istio/test-infra
-    branch: master
+    branch: release-1.9
```
Comment (jacob-delgado) [inline example/manifest.yaml:50]: change to `master`

File: release/build.sh [hunk 2]
```diff
@@ -62,17 +62,17 @@ ${DEPENDENCIES:-$(cat <<EOD
...
+    branch: release-1.9
     goversionenabled: true
   gogo-genproto:
     git: https://github.com/istio/gogo-genproto
-    branch: master
+    branch: release-1.9
   test-infra:
     git: https://github.com/istio/test-infra
-    branch: master
+    branch: release-1.9
```
Comment (jacob-delgado) [inline release/build.sh:72]: change to `master`

File: test/publish.sh [hunk 3]
```diff
@@ -54,17 +54,17 @@ dependencies:
...
+    branch: release-1.9
     goversionenabled: true
   gogo-genproto:
     git: https://github.com/istio/gogo-genproto
-    branch: master
+    branch: release-1.9
   test-infra:
     git: https://github.com/istio/test-infra
-    branch: master
+    branch: release-1.9
```
Comment (jacob-delgado) [inline test/publish.sh:64]: change to `master`

File: example/manifest.yaml (hunk 1 above)
Comment (shamsher31) [reply example/manifest.yaml]: Done.

File: release/build.sh (hunk 2 above)
Comment (shamsher31) [reply release/build.sh]: Done.

File: test/publish.sh (hunk 3 above)
Comment (shamsher31) [reply test/publish.sh]: Done.

**Review by jacob-delgado — APPROVED** [review]

**Pull request merged**